dataprofiler/profilers/profiler_options.py

```diff
@@ -164,25 +164,26 @@ def _validate_helper(self, variable_path='HistogramOption'):
         :rtype: list(str)
         """
         errors = super()._validate_helper(variable_path=variable_path)
-        if self.method is not None:
-            valid_methods = ['auto', 'fd', 'doane', 'scott',
-                             'rice', 'sturges', 'sqrt']
-            str_list_check = False
-            if isinstance(self.method, list):
-                str_list_check = all([isinstance(item, str)
-                                      for item in self.method])
-            if not isinstance(self.method, str) and not str_list_check:
-                errors.append("{}.method must be a string or list of strings "
-                              "from the following: {}."
-                              .format(variable_path, valid_methods))
-            if isinstance(self.method, str) and self.method not in valid_methods:
-                errors.append("{}.method must be one of the following: {}."
-                              .format(variable_path, valid_methods))
-            if isinstance(self.method, list) and \
-                    not set(self.method).issubset(set(valid_methods)):
-                errors.append("{}.method must be a subset of {}."
-                              .format(variable_path, valid_methods))
 
+        if self.bin_count_or_method is not None:
+            valid_methods = ['auto', 'fd', 'doane', 'scott', 'rice', 'sturges',
+                             'sqrt']
+            
+            value = self.bin_count_or_method
+            if isinstance(value, str):
+                value = [value]
+            if isinstance(value, int) and value >= 1:
+                pass  # use errors below if not a passing int
+            elif (not isinstance(value, list) or len(value) < 1
+                  or not all([isinstance(item, str) for item in value])):
+                errors.append("{}.bin_count_or_method must be an integer more "
+                              "than 1, a string, or list of strings from the "
+                              "following: {}.".format(variable_path,
+                                                      valid_methods))
+            elif not set(value).issubset(set(valid_methods)):
+                errors.append("{}.bin_count_or_method must be a subset or "
+                              "selection from: {}.".format(variable_path,
```

Comment:
This doesn't account for `bin_count_or_method` being able to be an integer, and my be useful for the user to know. Should the error message be the same for all cases, or should this one just be extended to say "if providing a bin method name, it must be a subset ..."